src/main/java/org/jenkinsci/plugins/saml/SamlAuthenticationToken.java

```diff
@@ -17,22 +17,23 @@
 
 package org.jenkinsci.plugins.saml;
 
-import org.acegisecurity.GrantedAuthority;
 import org.acegisecurity.providers.AbstractAuthenticationToken;
 
 public class SamlAuthenticationToken extends AbstractAuthenticationToken {
 
   private static final long serialVersionUID = 1L;
```

Comment:
As I know this makes sense only for classes that are stored/serialized somewhere. I'm not an expert in Jenkins but I almost sure that tokens are kind of dynamic objects and they are not stored for long time anywhere.